### PR TITLE
community/team: update

### PR DESCRIPTION
**Pull request by liewegas**
Missing a few bios.

Signed-off-by: Sage Weil <sage@newdream.net>

### CODE REVIEW

**Comment by sebastian-philipp** [comment]
the bottom half listing all the details of the team is also split up in "Core Team" and "Maintainers". do you want to shuffle those people around or drop the sub-sections?

File: src/en/community/team/index.html [hunk 1]
```diff
@@ -262,26 +226,35 @@ <h3 class="h3" id="joao-luis">Joao Luis</h3>
...
-  </div>
-</section>
-
-<section class="section">
-  <div class="max-w-256">
-    <h2 class="h2 mb-12 lg:mb-16">Maintainers</h2>
 
     <div class="pt-0 section">
       <div class="grid md:grid--cols-56-auto lg:grid--cols-72-auto">
-        <img alt="Zheng Yan" class="rounded-2 to-md:max-w-56 w-full" loading="lazy" src="/assets/bitmaps/photo-zheng-yan.jpg" />
```
Comment (batrick) [inline src/en/community/team/index.html:274]: Jeff Layton is now the maintainer for `fs/ceph`.

**Comment by liewegas** [comment]
Ok removed the dead links for everyone who is missing a bio.  I think we can merge this then and leave adding those bios as a TODO.

Unless @jtlayton @idryomov @zdover23 want to chime in with some content!

**Comment by zdover23** [comment]
I'll come up with something this weekend.

**Review by batrick — APPROVED** [review]

**Review by sebastian-philipp — APPROVED** [review]

**Review by jtlayton — APPROVED** [review]

**Comment by sebastian-philipp** [comment]
can we get this in? This already a big improvement to the existing page

**Comment by jtlayton** [comment]
I queued up a PR with a photo in https://github.com/ceph/ceph.io/pull/258.

**Comment by zdover23** [comment]
Merge this one without me; I'm hip-deep in Nomad docs rewrites.

**Review by jdurgin — APPROVED** [review]

**Pull request merged**